### PR TITLE
V2 of Entity Explorer - Account notebook

### PR DESCRIPTION
**Pull request by ianhelle**

### CODE REVIEW

**Comment by zhzhao8888** [comment]
1. If this going to Template gallery, you may need to remove output results if they contain PII or secrets like data, which will fail the build check.
2. If this is going to Template, then please update notebookmetadada.json file as well.

File: Entity Explorer - Account.ipynb [hunk 1]
```diff
@@ -6,20 +6,23 @@
    "source": [
```
Comment (petebryan) [inline Entity Explorer - Account.ipynb:6]: <p>Not sure where the Path import is used.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/142/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (petebryan) [inline Entity Explorer - Account.ipynb:6]: <p>Some markdown issues in this cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/142/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (petebryan) [inline Entity Explorer - Account.ipynb:6]: <p>I find this a bit confusing as its not clear immediately what the difference between "AzureActiveDirectory" and "OfficeActivty" is here - Account types doesn't adequately describe what this is. I would consider adding some more markdown that explains this.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/142/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (petebryan) [inline Entity Explorer - Account.ipynb:6]: <p>Un-needed cell?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/142/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Comment (petebryan) [inline Entity Explorer - Account.ipynb:6]: <p>DatFrame &gt; DataFrame</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/142/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Comment (petebryan) [inline Entity Explorer - Account.ipynb:6]: <p>Some markdown issues here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/142/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

**Comment by ianhelle** [comment]
This is included in #146

**Pull request closed without merging**